module/os/linux/zfs/zfs_vnops_os.c

```diff
@@ -1668,7 +1673,11 @@ zfs_getattr_fast(struct inode *ip, struct kstat *sp)
 
 	mutex_enter(&zp->z_lock);
 
+#ifdef HAVE_GENERIC_FILLATTR_USERNS
+	generic_fillattr(user_ns, ip, sp);
+#else
 	generic_fillattr(ip, sp);
+#endif
```

Comment:
Perhaps provide a `zpl_generic_fillattr(user_ns, ip, sp);` wrapper in `vfs_compat.h` which discards the `user_ns`in the `!HAVE_GENERIC_FILLATTR_USERNS` case.

Reply:
Let me know what you thing about the solution in 04d4c20

Reply:
Err 58e2ab2, because I missed the change in the header.